tutorials/editor/using_the_web_editor.rst

```diff
@@ -0,0 +1,120 @@
+.. _doc_using_the_web_editor:
+
+Using the Web editor
+====================
+
+Since Godot 3.2.4, there is a `Web editor <https://editor.godotengine.org/>`__
+you can use to work on new or existing projects.
+
+.. note::
+
+    The web editor is in a preliminary stage. While its feature set may be
+    sufficient for educational purposes, it is currently **not recommended for
+    production work**. See :ref:`doc_using_the_web_editor_limitations` below.
+
+Browser support
+---------------
+
+The Web editor requires support for WebAssembly's SharedArrayBuffer. This
+is in turn required to support threading in the browser. The following desktop
+browsers support WebAssembly threading and can therefore run the web editor:
+
+- Chrome 68 or later
+- Firefox 79 or later
+- Edge 79 or later
+
+Opera and Safari are not supported yet. Safari may work in the future once
+proper threading support is added.
+
+**Mobile browsers are currently not supported.**
+
+The web editor supports both the GLES3 and GLES2 renderers, although GLES2 is
+recommended for better performance and compatibility with old/low-end hardware.
+
+.. note::
+
+    If you use Linux, due to
+    `poor Firefox WebGL performance <https://bugzilla.mozilla.org/show_bug.cgi?id=1010527>`__,
+    it's recommended to use a Chromium-based browser instead of Firefox.
```

Comment:
honestly, I don't really like favouring one browser here.
At least in my experiences FF is performing quite okay with the editor.
That said, Chrome do have some nice features that FF is missing, e.g. better clipboard API so I don't mind to much.

Reply:
> At least in my experiences FF is performing quite okay with the editor.

The web editor is certainly usable on Firefox, but Chromium-based browsers will pretty much always perform better on Linux for anything WebGL. This is especially true at larger resolutions (since the compositing performance largely depends o nresolution).